Packet++/src/GvcpLayer.cpp

```diff
@@ -0,0 +1,101 @@
+#define LOG_MODULE PacketLogModuleGvcpLayer
+
+#include "GvcpLayer.h"
+#include <cstring>
+
+/**
+ * @file GvcpLayer.cpp
+ * @author An-Chi Liu (phy.tiger@gmail.com)
+ */
+
+namespace pcpp
+{
+
+	std::ostream& operator<<(std::ostream& os, GvcpCommand command)
+	{
+		os << "0x" << std::hex << static_cast<uint16_t>(command) << std::dec;
+		return os;
+	}
+
+	std::ostream& operator<<(std::ostream& os, GvcpResponseStatus status)
+	{
+		os << "0x" << std::hex << static_cast<uint16_t>(status) << std::dec;
+		return os;
+	}
+
+	/*---------------------- Class GvcpLayer ----------------------------*/
+
+	GvcpLayer::GvcpLayer(uint8_t* data, size_t dataSize, Layer* prevLayer, Packet* packet)
+	    : Layer(data, dataSize, prevLayer, packet)
+	{
+		m_Protocol = Gvcp;
```

Comment:
Hmm... we should probably add an overload to `Layer::Layer(...)` that accepts a protocol parameter too, instead of needing to be set manually afterwards.

Reply:
True, we should do the refactoring for this issue later.

Reply:
Opened a separate issue for this.